docs/USING_FUNCTIONS.md

```diff
@@ -109,7 +111,7 @@ For example if the `package.json` looks like this:
 
 You'll want to create a dependency entry with the name `stripe` and the version `^8.20.0`.
 
-![Screenshot of adding dependencies](images/add_dependencies.png)
+![Screenshot of adding dependencies](images/add-dependencies-v2.png)
```

Comment:
For some reason the text always talked about "stripe" as a dependency, but the screenshot showed "lodash". If this is getting an update, then I think it would be great to match the prose to the image.